spec/system/check_records/user_searches_for_teacher_with_restrictions_spec.rb

```diff
@@ -30,6 +30,7 @@ def and_search_returns_a_restricted_record
   def then_i_see_the_restriction_on_the_result
     expect(page).to have_title("Search results (1) - Check a teacher’s record")
     expect(page).to have_content("RESTRICTIONS")
+    expect(page).to have_content("Restrictions\t\nFailed induction\nProhibition by the General Teaching Council England (GTCE)")
```

Comment:
Minor feedback: asserting on the presence of whitespace like this can be flaky, due to minor rendering variations across platforms or library versions.

An alternative might be to scope the Restrictions summary row using `within`, retrieving all the `li` tags, and then making assertions about their content.